aws/data_source_aws_secretsmanager_secret_version.go

```diff
@@ -14,6 +14,10 @@ func dataSourceAwsSecretsManagerSecretVersion() *schema.Resource {
 		Read: dataSourceAwsSecretsManagerSecretVersionRead,
 
 		Schema: map[string]*schema.Schema{
+			"arn": &schema.Schema{
```

Comment:
Nit: the `&schema.Schema` here is extraneous since Go 1.7 as its previously defined by the `map[string]*schema.Schema` above